code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -957,8 +960,8 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	if(HAS_TRAIT(target, TRAIT_BRAWLING_KNOCKDOWN_BLOCKED) || target.stat == DEAD) //If our target is dead or has specailized armor, there is no way to inflict these effects.
 		return
 
-	// If our target is staggered, the target's armor, minus our limb effectiveness sets the minimum necessary amount of damage sustained to cause an effect. Minimum 40, max 200 for sanity reasons
-	if(staggered && (target.getFireLoss()*0.5 + target.getBruteLoss()*0.5) >= min(armor_block - limb_accuracy, 40, 200))
+	// If our target is staggered, the target's armor, minus our limb effectiveness sets the minimum necessary amount of damage sustained to cause an effect. We clamp the value for sanity reasons.
+	if(staggered && target_brute_and_burn >= clamp(((armor_block && armor_block > UNARMED_COMBO_HIT_HEALTH_BASE) ? armor_block : UNARMED_COMBO_HIT_HEALTH_BASE) - limb_accuracy, 0, 200))
```

Comment:
This ternary makes it a bit hard to read, I'd separate it out into a var.
And from what I can tell, that ternary is just trying to do a `max`? (if armor block > some define: use armor block, else: use some define)
```dm
var/effective_armor = max(armor_block, UNARMED_COMBO_HIT_HEALTH_BASE) - limb_accuracy
if(staggered && target_brute_and_burn >= clamp(effective_armor, 0, 200)
```